### PR TITLE
test: new field `Incompatible` of EngineImage

### PR DESCRIPTION
**Pull request by mantissahz**
#### Which issue(s) this PR fixes:

Issue longhorn/longhorn#5842, longhorn/longhorn#7539

#### What this PR does / why we need it:

Modify the manual test case for engine version enforcement.

### CODE REVIEW

File: docs/content/manual/release-specific/v1.6.0/test-engine-version-enforcement.md [hunk 1]
```diff
@@ -24,19 +25,19 @@ And should see incompatible version error in longhorn-manager Pod logs
...
+**When** downgraded Longhorn to v1.5.x
 **Then** Longhorn components should be running
 
-**When** upgraded v1.4.1 volume (volume-1) engine  
+**When** upgraded v1.4.1 volume (volume-1) engine
 And upgraded Longhorn to v1.6.0
-**Then** Longhorn components should be running  
-And v1.4.x EngineImage state should be incompatible
+**Then** Longhorn components should be running
+And v1.4.x EngineImage state should be still deployed but status.incompatible should be true.
```
Comment (c3y1huang) [inline docs/content/manual/release-specific/v1.6.0/test-engine-version-enforcement.md:34]:
```suggestion
And v1.4.x EngineImage state should be deployed and incompatible should be true.
```

**Review by c3y1huang — CHANGES_REQUESTED** [review]
The trailing spaces are deliberately placed for line breaks.

It's looking like this without the spaces.
Screenshot from 2024-01-18 12-04-38

File: docs/content/manual/release-specific/v1.6.0/test-engine-version-enforcement.md (hunk 1 above)
Comment (mantissahz) [reply docs/content/manual/release-specific/v1.6.0/test-engine-version-enforcement.md]: Modified the description and added the spaces back. Thanks.

**Review by yangchiu — APPROVED** [review]
LGTM

**Review by c3y1huang — COMMENTED** [review]
LGTM, however, I noticed that the commit is signed off with a Gmail account instead of the one from SUSE. If this is intended then we could merge this. @mantissahz could you confirm?

**Comment by mantissahz** [comment]
>LGTM, however, I noticed that the commit is signed off with a Gmail account instead of the one from SUSE. If this is intended then we could merge this. @mantissahz could you confirm?

Corrected.
Thank you, @c3y1huang, and it should be the one from SUSE.

**Review by c3y1huang — APPROVED** [review]

**Pull request merged**